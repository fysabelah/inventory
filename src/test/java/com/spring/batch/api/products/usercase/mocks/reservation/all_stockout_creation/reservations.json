[
  {
    "sku": "BOOKS-78598-9898",
    "quantity": 9
  },
  {
    "sku": "CLOTHES-SOME-JEANS-JEANS-BRAND-YELLOW-P",
    "quantity": 7
  },
  {
    "sku": "CLOTHES-SOME-JEANS-JEANS-BRAND-YELLOW-PP",
    "quantity": 11
  },
  {
    "sku": "ELECTRONICS-SOME-TV-TV-BRAND-TV-YELLOW",
    "quantity": 4
  },
  {
    "sku": "ELECTRONICS-SOME-TV-TV-BRAND-TV-RED",
    "quantity": 5
  },
  {
    "sku": "SHOES-BOOTS-FOR-PRETTY-GIRLS-SHOE-BRAND-BLACK-38",
    "quantity": 11
  },
  {
    "sku": "SHOES-BOOTS-FOR-PRETTY-GIRLS-SHOE-BRAND-BLACK-40",
    "quantity": 26
  }
]