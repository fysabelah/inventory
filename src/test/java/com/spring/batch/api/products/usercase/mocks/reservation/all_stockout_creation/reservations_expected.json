[
  {
    "id": null,
    "sku": "BOOKS-78598-9898",
    "quantity": 9,
    "status": "STOCKOUT",
    "date": "2024-07-01T08:40:00Z"
  },
  {
    "id": null,
    "sku": "CLOTHES-SOME-JEANS-JEANS-BRAND-YELLOW-P",
    "quantity": 7,
    "status": "STOCKOUT",
    "date": "2024-07-01T08:40:00Z"
  },
  {
    "id": null,
    "sku": "CLOTHES-SOME-JEANS-JEANS-BRAND-YELLOW-PP",
    "quantity": 11,
    "status": "STOCKOUT",
    "date": "2024-07-01T08:40:00Z"
  },
  {
    "id": null,
    "sku": "ELECTRONICS-SOME-TV-TV-BRAND-TV-YELLOW",
    "quantity": 4,
    "status": "STOCKOUT",
    "date": "2024-07-01T08:40:00Z"
  },
  {
    "id": null,
    "sku": "ELECTRONICS-SOME-TV-TV-BRAND-TV-RED",
    "quantity": 5,
    "status": "STOCKOUT",
    "date": "2024-07-01T08:40:00Z"
  },
  {
    "id": null,
    "sku": "SHOES-BOOTS-FOR-PRETTY-GIRLS-SHOE-BRAND-BLACK-38",
    "quantity": 11,
    "status": "STOCKOUT",
    "date": "2024-07-01T08:40:00Z"
  },
  {
    "id": null,
    "sku": "SHOES-BOOTS-FOR-PRETTY-GIRLS-SHOE-BRAND-BLACK-40",
    "quantity": 26,
    "status": "STOCKOUT",
    "date": "2024-07-01T08:40:00Z"
  }
]